File: DDS.Client/ClientApp/src/app/models/ownership.ts
import { Game } from "./game";
import { User } from "./user";

export class Ownership {

  constructor(
    public ownershipId?: number,
    public userId?: number,
    public gameId?: number,
    public date?: Date,
    public price?: number,
    public game?: Game,
    public user?: User
  ) {

  }
}

File: DDS.WebApi/Models/Ownership.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

#nullable disable

namespace DDS.WebApi.Models
{
    public partial class Ownership
    {
        public Ownership()
        {
            Reviews = new HashSet<Review>();
        }

        public int OwnershipId { get; set; }
        public int UserId { get; set; }
        public int GameId { get; set; }
        public DateTime Date { get; set; }
        public decimal Price { get; set; }

        public virtual Game Game { get; set; }
        public virtual User User { get; set; }
        [JsonIgnore]
        public virtual ICollection<Review> Reviews { get; set; }
    }
}
